go/enclave/db/storage.go

```diff
@@ -685,6 +688,15 @@ func (s *storageImpl) FilterLogs(
 	return s.loadLogs(requestingAccount, query, queryParams)
 }
 
+func (s *storageImpl) GetContractCount() (*big.Int, error) {
+	count, err := s.db.Get([]byte("contractCreationCountKey"))
```

Comment:
Feels weird that the key string is duplicated here, is there a way to call the public `ReadContractCreationCount()` method? If not then maybe it should be a public constant somewhere.